.github/workflows/e2e.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     branches:
       - changeset-release/main
+    paths:
+      - packages/wrangler/e2e/**
```

Comment:
Is this enough? It feels like this actually restricts the job to run only on pushes that match both the branch and the path?

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore:~:text=the%20path%20again.-,If%20you%20define%20both%20branches/branches%2Dignore%20and%20paths/paths%2Dignore%2C%20the%20workflow%20will%20only%20run%20when%20both%20filters%20are%20satisfied.,-The%20paths%20and

Reply:
Good catch! Any idea how to make it run if either match?

Trying to make "push:" an array so I can specify the same event twice here https://github.com/cloudflare/workers-sdk/pull/5793/commits/0daaaf008ca205bbf36b37cac03a3e60c35b8f71

Reply:
I've switched strategy. Rather than mess with the existing job, I've added a job to automatically add the 'e2e' label if files within the dir change (this feature comes out-of-the-box with actions/labeler)